packages/api-reference/src/components/SidebarElement.vue

```diff
@@ -94,6 +95,9 @@ defineExpose({ el })
           class="sidebar-icon"
           :src="item.icon.src" />
         <p>{{ item.title }}</p>
+        <div
+          class="sidebar-heading-type"
+          :class="`sidebar-heading-type--${item.httpVerb}`"></div>
```

Comment:
Am on mobile, maybe I’ve overseen it, but should we add this class only when httpVerb is set?

Reply:
💯% 

Will update